src/Utilities/Utilities.jl

```diff
@@ -114,11 +114,11 @@ parsed from.
 The keyword argument `skip = N` drops the leading `N` lines from the input string.
 """
 function parseblock(code::AbstractString, doc, page; skip = 0, keywords = true)
+    # work only with LF line endings internally
+    code = replace(code, "\r\n" => "\n")
     # Drop `skip` leading lines from the code block. Needed for deprecated `{docs}` syntax.
     code = string(code, '\n')
     code = last(split(code, '\n', limit = skip + 1))
-    # Check whether we have windows-style line endings.
-    offset = occursin("\n\r", code) ? 2 : 1
```

Comment:
I suspect the error was from this beeing `"\n\r"` and not `"\r\n"`, but I figured we might as well just use `"\n"` internally.

Reply:
And also, this local `code` argument does not leak out of this parser function anyways.